### ISSUE TITLE
Better Design and Layout for Profile Page

### ISSUE THREAD

**Issue by ajwad-shaikh** [ajwad-shaikh/BusKaro#7]
**Is your feature request related to a problem? Please describe.**
No, not related to a problem. It just looks aesthetically unpleasant.

**Describe the solution you'd like**
A more spread out page design with more options for the user.

**Describe alternatives you've considered**
This could move to a permanent sidebar on the dashboard if we do not need a separate page.

**Additional context**

![image](https://user-images.githubusercontent.com/37571185/66000407-ffe9ae80-e4bc-11e9-8ad3-60961173c16f.png)

**Comment by mcastre** [comment]
Hey i can help with this if you'd like!

**Comment by ajwad-shaikh** [comment]
Sure @mcastre 

Assigning you

**Comment by mcastre** [comment]
@ajwad-shaikh is it possible to run the app using MAMP?

**Comment by ajwad-shaikh** [comment]
@mcastre Do raise an issue if something breaks on MAMP.

**Comment by mcastre** [comment]
@ajwad-shaikh im getting this error:
`Connected unsuccessfullyConnection failed: Access denied for user 'root'@'localhost' (using password: NO)`
after installing the repo in the MAMP htdocs folder and running the server. Have you encountered this issue before?

**Comment by vardanabhanot** [comment]
hi sir, so do you want a complete redesign of the profile page or just minor changes and just found you are not using best practices in the design

**Comment by ajwad-shaikh** [comment]
Hey @VardanBhanot, cut the salutations, welcome to open source!
It's great that you pointed that out. A complete redesign would be ideal. 
I hate submission dates but I had one so had to rush with something that could fill the page. Anyway, do let me know what you think would be best.

Welcome! :)

**Comment by vardanabhanot** [comment]
> Hey @VardanBhanot, cut the salutations, welcome to open source!
> It's great that you pointed that out. A complete redesign would be ideal.
> I hate submission dates but I had one so had to rush with something that could fill the page. Anyway, do let me know what you think would be best.
> 
> Welcome! :)

may I know whats your deadline for this project?
mainly i will focus on making this page clean and interactive adding shadows and using illustrations here or maybe.

**Comment by ajwad-shaikh** [comment]
That sounds great. There is no deadline as such but I would be glad if this overhaul is done before the end of this year @VardanBhanot

**Comment by vardanabhanot** [comment]
yes sure it will take me a few days to do that, just i have few deadlines i will do after that but i will do it by end of oct @ajwad-shaikh
